src/fontra/client/web-components/menu-panel.js

```diff
@@ -266,8 +266,6 @@ export class MenuPanel extends SimpleElement {
   }
 
   handleKeyDown(event) {
-    event.stopImmediatePropagation();
```

Comment:
Thanks for pointing that out. I will have to play with that. There may be a reason why that's there, but I don't know off the top of my head.